forms-flow-web-root-config/sample.env

```diff
@@ -21,10 +21,10 @@
 #MF_FORMSFLOW_THEME_URL=https://s3.ap-northeast-1.amazonaws.com/formsflow.ai-micro-front-ends/forms-flow-theme/forms-flow-theme.gz.js
 
 #TODO only for Customised Local delevelopment with code from https://github.com/AOT-Technologies/forms-flow-ai-micro-front-ends  use below default values instead
-#MF_FORMSFLOW_NAV_URL=//{your-ip-address}:8080/forms-flow-nav.js
-#MF_FORMSFLOW_ADMIN_URL=//{your-ip-address}:8001/forms-flow-admin.js
-#MMF_FORMSFLOW_SERVICE_URL=://{your-ip-address}:8081/forms-flow-service.js
-#MF_FORMSFLOW_THEME_URL=//{your-ip-address}:8082/forms-flow-theme.js
+#MF_FORMSFLOW_NAV_URL=//{your-ip-address}:3005/forms-flow-nav.js
```

Comment:
Need to modify the ports in the sample.env local reference

Reply:
Yes this is sample.env

Reply:
ok Got it